src/functional/ffront/field_operator_ast.py

```diff
@@ -54,7 +54,7 @@ class Symbol(eve.GenericNode, LocatedNode, Generic[SymbolT]):
 
 
 class Expr(LocatedNode):
-    type: Optional[common_types.SymbolType] = None  # noqa A003
+    type: common_types.SymbolType = common_types.DeferredSymbolType(constraint=None)  # noqa A003
```

Comment:
These have always been treated as semantically equivalent. Deferred is just a little easier to work with.